cmd/omegaup-grader/frontend_handler.go

```diff
@@ -320,6 +327,36 @@ func broadcastRun(
 	if err != nil {
 		return err
 	}
+	var groupName string
+	var scoreByGroup float64
+	rows, error := queryWithRetry(
+		db,
+		`SELECT
+			rg.group_name, rg.score
+		FROM
+			Runs_Groups rg
+		WHERE
+			rg.run_id = ?;`, run.ID)
+	if error != nil {
+		return error
+	}
+
+	var scores map[string]float64
+	scores = make(map[string]float64)
+	for rows.Next() {
+		err = rows.Scan(&groupName, &scoreByGroup)
+		if err != nil {
+			ctx.Log.Error(
+				"Failed to get run",
+				map[string]any{
+					"err": err,
+				},
+			)
+			continue
```

Comment:
```suggestion
			return fmt.Errorf("failed to scan run group: %w", err)
```

esto suena a que no debería continuar si falla: si continuamos podemos regresar información incorrecta.

Reply:
Listo